www/content/workstation/config_rb.md

```diff
@@ -28,16 +28,16 @@ A `config.rb` file is used to specify configuration details for knife.
 
 A `config.rb` file:
 
--   Is loaded every time the knife executable is run
--   The default location at which Chef Infra Client expects to find this
+- Is loaded every time the knife executable is run
+- The default location at which Chef Infra Client expects to find this
     file is `~/.chef/config.rb`; use the `--config` option from the
     command line to change this location
--   Is not created by default
--   Is located by default at `~/.chef/config.rb` (UNIX and Linux
+- Is not created by default
+- Is located by default at `~/.chef/config.rb` (UNIX and Linux
```

Comment:
```suggestion
- Is located by default at `~/.chef/config.rb` (macOS and Linux
```

Make's it more clear for macOS users. We don't support BSD or Solaris on Workstation so this is fine.